app/views/transaction_detail/_details.njk

```diff
@@ -49,5 +49,12 @@
     <td>GOV.UK payment ID:</td>
     <td id="payment-id">{{charge_id}}</td>
   </tr>
+
+  {% if delayed_capture %}
+  <tr>
+    <td>Delayed capture:</td>
+    <td id="delayed-capture-id">On</td>
```

Comment:
I think this should just be `delayed-capture` (the other things with `-id` actually are IDs — `pyament-id`, `transaction_id` — but this isn’t)